spec/requests/admin/legal_aid_applications_spec.rb

```diff
@@ -37,6 +37,20 @@
     end
   end
 
+  describe 'PATCH /admin/legal_aid_applications/create_test_applications' do
+    subject { patch create_test_applications_admin_legal_aid_applications_path }
+    let(:count) { 1 }
+
+    it 'creates test legal_aid_applications' do
+      expect { subject }.to change { LegalAidApplication.count }.to be > 1
```

Comment:
Why are you being so vague here - you know exactly how many new items should be created:
```
number_new = TestApplicationCreationService::APPLICATION_TEST_TRAITS.size
expect { subject }.to change { LegalAidApplication.count }.by(number_new)
```